tests/common.py

```diff
@@ -376,6 +379,10 @@ def clear_instance(event):
     finally:
         # Restore timezone, it is set when creating the hass object
         dt_util.set_default_time_zone(orig_tz)
+        # Remove loop shutdown indicator to not interfere with additional hass objects
+        loop = asyncio.get_running_loop()
+        with suppress(AttributeError):
+            delattr(loop, _SHUTDOWN_RUN_CALLBACK_THREADSAFE)
```

Comment:
This fix should probably be a separate PR.

Can we operate on `hass.loop` instead?